components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java

```diff
@@ -150,6 +153,9 @@ public void updateConfiguration(String tenantDomain, Map<String, String> configu
             ConnectorConfig config = new ConnectorConfig();
             Map<String, String> propertyFriendlyNames = list.get(i).getPropertyNameMapping();
             Map<String, String> propertyDescriptions = list.get(i).getPropertyDescriptionMapping();
+
```

Comment:
Let's add a comment and remove unnecessary new lines. WDYT?

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767